core/src/main/java/hudson/model/PasswordParameterDefinition.java

```diff
@@ -54,6 +54,7 @@ public PasswordParameterDefinition(String name, String defaultValue, String desc
         this.defaultValue = Secret.fromString(defaultValue);
     }
 
+    // TODO consider switching @DataBoundConstructor to a PasswordParameterDefinition(String) overload
```

Comment:
Looked tricky due to the historical confusion over the attribute name: `defaultValue` vs. `defaultValueAsSecret`.